packages/wrangler/src/__tests__/hyperdrive.test.ts

```diff
@@ -402,7 +437,42 @@ describe("hyperdrive commands", () => {
 		`);
 		expect(std.out).toMatchInlineSnapshot(`
 			"🚧 Creating 'test123'
-			✅ Created new Hyperdrive config: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+			✅ Created new Hyperdrive PostgreSQL config: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+			📋 To start using your config from a Worker, add the following binding configuration to your Wrangler configuration file:
+
+			{
+			  \\"hyperdrive\\": [
+			    {
+			      \\"binding\\": \\"HYPERDRIVE\\",
+			      \\"id\\": \\"xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx\\"
+			    }
+			  ]
+			}"
+		`);
+	});
+
+	it("should create a hyperdrive config given individual params instead of a connection string", async () => {
+		const reqProm = mockHyperdriveCreate();
+		await runWrangler(
+			"hyperdrive create test123 --host=example.com --database=neondb --user=test --password=password --port=1234 --origin-scheme=mysql"
```

Comment:
The `origin-scheme` flag is currently set here: https://github.com/cloudflare/workers-sdk/blob/1320f77725b8ee4d2f8ead16287428026c047ae6/packages/wrangler/src/hyperdrive/create.ts#L19

I think it should be moved to the `upsertOptions` function in `index.ts` and be given an alias to just `scheme` for consistency. We could also consider removing the default.

Reply:
It's already present in `upsertOptions` with the requested alias